docs/ops.html

```diff
@@ -4054,16 +4054,18 @@ <h3>Reverting to ZooKeeper mode During the Migration</h3>
           </ul>
         </td>
         <td>
-          It is important to perform the <code>zookeeper-shell.sh</code> step quickly, to minimize the amount of
-          time that the cluster lacks a controller.
+          It is important to perform the <code>zookeeper-shell.sh</code> step <b>quickly</b>, to minimize the amount of
+          time that the cluster lacks a controller. Until the <code> /controller</code> znode is deleted,
+          you can also ignore any broker errors in the log about controller missing.
```

Comment:
> Until the <code> /controller</code> znode is deleted, you can also ignore any broker errors in the log about controller missing.

Maybe:
`You can also ignore any errors in the broker log about failing to connect to the Kraft controller. Those error logs should disappear after second roll to pure zookeeper mode.`

WDYT?

Reply:
I was avoiding too much text, but of course this one is clearer. I will update it.

Reply:
Done.